echo/test/Echo.js

```diff
@@ -34,23 +34,35 @@ describe("Echo contract", async function () {
                 this.timeout(50000);
 
                 it("should update the echo variable", async function () {
-                        await instance.scream("Hello World!");
+                        await instance.scream("Hello World!", {
+                                gasPrice: ethParams.txGasPrice,
+                                gasLimit: ethParams.txGasLimit
+                        });
```

Comment:
Changes like this warrant changes in the respective `README` files. We should finalize our decision on how to proceed and then either remove the gas parameters or update the `READMEs `as well as the scripts.